src/fsharp/vs/ServiceAssemblyContent.fs

```diff
@@ -145,6 +145,7 @@ type internal RawEntity =
       IsPublic: bool
       TopRequireQualifiedAccessParent: Idents option
       AutoOpenParent: Idents option
+      Item: NameResolution.Item
```

Comment:
My gut feeling is that this "RawEntity" type should disappear in favour of the FSharpSymbol/FSharpEntity type, which is designed to be a public FCS API.  Up to now I've assumed RawEntity wouldn't reveal any TAST data structures - however now it's carrying an Item, which contains TAST items.  I think we need to look closely at this, as direct access to TAST data is really not something we want outside the compiler itself.